mixer/pkg/adapter/standardTypes.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adapter
+
+// Strong Go types for template-specific instance fields that are sent to the adapters.
+type (
+	// DNSName is associated with template field type istio.mixer.v1.template.DNSName
+	DNSName string
+	// EmailAddress is associated with template field type istio.mixer.v1.template.EmailAddress
+	EmailAddress string
+	// URI is associated with template field type istio.mixer.v1.template.Uri
+	URI string
```

Comment:
Can we use url.URL?

Reply:
As discussed offline, we should do this but it needs work on the evaluator. For now I have marked the corresponding types inside mixer/v1/template/standard_types.proto as DO NOT USE. I will leave the rest of the code as is so that we have the tests and everything ready to be used once the evaluator code is ready.

Reply:
As discussed offline, we should do this but it needs work on the evaluator. For now I have marked the corresponding types inside mixer/v1/template/standard_types.proto as DO NOT USE. I will leave the rest of the code as is so that we have the tests and everything ready to be used once the evaluator code is ready.